src/app/tab-events/tab-events.page.html

```diff
@@ -0,0 +1,92 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-title>Eventos</ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <ng-container *ngIf="majorEvents; else loading">
+    <h1 *ngIf="majorEvents.length === 0" class="ion-text-center">Nenhum evento encontrado</h1>
+
+    <div *ngIf="majorEvents.length > 0" class="event-container">
+      <ng-container *ngFor="let event of majorEvents">
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>{{event.name}}</ion-card-title>
+          </ion-card-header>
+
+          <ion-card-content>
+            <p>{{event.description}}</p>
+          </ion-card-content>
+
+          <div class="card-section">
+            <div>
+              <ion-icon name="calendar-outline"></ion-icon>
+              Datas
+            </div>
+
+            <ion-item lines="none">
+              Evento
+              <span slot="end">
+                {{ getDateFromTimestamp(event.dateStart) | date: 'dd/MM' }}
+                {{ event.dateEnd && '- ' + (getDateFromTimestamp(event.dateEnd) | date: 'dd/MM') }}
+              </span>
+            </ion-item>
+
+            <ion-item lines="none">
+              Inscrição
+              <span slot="end">
+                {{ getDateFromTimestamp(event.subscriptionDateStart) | date: 'dd/MM' }}
+                {{ event.subscriptionDateEnd && '- ' + (getDateFromTimestamp(event.subscriptionDateEnd) | date: 'dd/MM') }}
+              </span>
+            </ion-item>
+          </div>
+
+          <div class="card-section">
+            <div>
+              <ion-icon name="wallet-outline"></ion-icon>
+              Valores
+            </div>
+
+            <ng-container *ngIf="isObject(event.price); then multiPrice else uniquePrice"></ng-container>
+
+            <ng-template #uniquePrice>
+              <ion-item lines="none">
+                Preço único
+                <span slot="end">{{ (event.price / 100) | currency:'BRL' }}</span>
+              </ion-item>
+            </ng-template>
+
+            <ng-template #multiPrice>
+              <ion-item lines="none">
+                Alunos da UNESP
+                <span slot="end">{{ (event.price.priceStudents / 100) | currency:'BRL' }}</span>
+              </ion-item>
+
+              <ion-item lines="none">
+                Alunos
```

Comment:
Se não ficar muito grande, prefiro
```suggestion
                Alunos de outras instituições
```

Penso que "outros alunos", "alunos de outras IES", etc tem entendimento prejudicado.